quotes.json

```
@@ -5473,6 +5473,12 @@
     "author": "Satchel Paige",
     "text": "Don't look back. Something might be gaining on you."
   },
+  {
+    "author": "Satya Nadella",
+    "source": "https://www.goodreads.com/author/quotes/15424914.Satya_Nadella",
```

Hello 👋 We should use the direct link to this specific quote as a source. 

```suggestion
    "source": "https://www.goodreads.com/quotes/8907834",
```